Rename Various Icons (v3)

PeterShaggyNoble opened the pull request:
**Issues:** see table below

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [ ] I optimized the icon with SVGO or SVGOMG
  - [ ] The SVG `viewbox` is `0 0 24 24`

### Description
Opening this draft now so we can start gathering up icons that need to be renamed in [ `v3`](https://github.com/simple-icons/simple-icons/milestone/2) in a single PR and close out a few issues. The branch on my fork is set up so that any other member of the @simple-icons/maintainers team can commit any additional renames directly to it.

~In this instance, I'd suggest when merging this that we do a merge commit for easier tracking.~ Scratch that, I made a mess of it!

### Affected Icons
| Old Name       | New Name           | Issue |
|----------------|--------------------|-------|
| Adobe Premiere | Adobe Premiere Pro | #2770 |
| Cassandra      | Apache Cassandra   | #2890 |
| Mojang         | Mojang Studios     | #3087 |
| Rails          | Ruby on Rails      | #2385 |
| Tik Tok        | TikTok             | #2766 |
| Zeit           | Vercel             | #2964 |

ericcornelissen commented:
Don't know if you noticed @PeterShaggyNoble, but [Vercel is out of order](https://travis-ci.com/github/simple-icons/simple-icons/jobs/321985144#L244) in the data file. It should come before `"Veritas"`.

PeterShaggyNoble commented:
Thanks, @ericcornelissen :+1: Had already spotted that but was holding off to see if any more renames were needed so as not to pollute the commits with fixes (although, I'm sure there'll be a bunch of conflicts to resolve by the time we get to v3!). I'll be fixing it with #3087.

PeterShaggyNoble commented:
Seeing as there will be a number of conflicts to resolve by the time we get to `v3`, I went ahead and renamed Mojang here, which includes the fix to the ordering of Vercel. I also updated the source URL to the press release linked in #3087.
